packages/snap-preact-components/src/components/Molecules/Result/Result.tsx

```diff
@@ -199,7 +228,8 @@ export const Result = observer((properties: ResultProps): JSX.Element => {
 });
 
 interface ResultSubProps {
-	badge: BadgeProps;
+	calloutBadge: CalloutBadgeProps;
```

Comment:
these should all be Partial<subprops>

Reply:
Left this as is. All other sub-prop interfaces don't have partial.